ccd-definition/ComplexTypes/C21OrderBundle.json

```diff
@@ -0,0 +1,34 @@
+[
```

Comment:
hmm this doesn't seem like a bundle to me... bundle sounds like it should be a collection of C21Order?

Reply:
do you have an idea for a better naming convention? I agree its messy - we build a 'C21Order' on page 1, then we build a 'JudgeAndLegalAdvisor' on page 2, and then upon submit I put both of these into a 'C21OrderBundle' which holds all info that needs to be displayed in the orders tab. it was essentially a hacky way to achieve entering judge details on 2nd page - the judge details couldn't be fields of the C21Order itself because you cant split that up over two pages in ccd (right?)

Reply:
could we use one object? And use caseEventToComplexTypes to hide the fields in the event, but leave them visible in the tabs?

Reply:
then you are just adding judge and legal advisor into your object

Reply:
do you mean add the judge stuff into the C21Order object and then use that for building the C21Orderbundle? we could do this (I believe noticeofproceedings does it like this), but we'd still need the separate judge object because we have 2 screens (I dont think theres any way around this, we need separate case fields for each page), and also it looks bad on the check your answers page since JudgeAndLegalAdvisor would be a collection inside C21Order (gets weirdly indented, maybe thats ok). thoughts?